website_cms/ROADMAP.rst

```diff
@@ -0,0 +1,16 @@
+Roadmap
+-------
+
+* Improve frontend forms (manage media, reuse backend forms)
+* Independent translations branches (skip odoo translation mechanism on demand via configuration)
+* Edit permissions control / sharing facilities / notifications, etc
+* Simplified interface for managing users and groups
+* Publication workflow management
+* Content history / versioning
+* Full text search using PG built-in search engine (see fts modules)
+* Shorter URLs (drop path ids for instance)
+* Performance fixes/tuning (use parent_left/right for instance)
+* Introduce portlets for sidebar elements
+* Add "collections" to fetch contents from the whole website (eg: "News from 2016")
+* Improve test coverage
+* Default theme
```

Comment:
Both are needed, @Yajo. A general one in the README of the repository to see the goal of it, and a specific roadmap inside the module.